### DISCUSSION TITLE
Welcome to DragonboundRe Discussions!

### DISCUSSION THREAD

**Discussion by Ox18** [alexwathever/SocialBound-HTML5#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Betololo** [comment]
Hi! I'm new to programing. I would like to know how to get the "formula" of the oblique throw used in Dragonbound. I was reading the JavaScript code, but I couldn't understand much. I'm very curious and dedicated, and getting to know how my favourite childhood game works is a dream of mine. Thank you!

**Comment by luizxd0** [comment]
Hi, I'm a python developer and I would like to support the project if possible.
Are you guys using any discord or other way of communicating?

**Reply by Betololo** [reply]
Hi! As I was studying python, I've decided to create an aimbot for Dragonbound, just to test my knowladge. My aimbot is already 100% working when there's no wind, but it still has a small error margin when there's wind (the stronger the wind, the greater the error margin). Maybe there's something wronf with my formula or with my code that I can't figure out... If you feel like helping me out with my aimbot project, it would be amazing! :) It would also be a good way for us to figure out the "formula" of the oblique throw used in the game. If you're interested, please let me know and I'll share with you my discord.

**Reply by luizxd0** [reply]
I could have a look at that and try to help you with the aimbot formula if you want. Share your discord please

**Reply by Betololo** [reply]
I've just created a discord server: https://discord.gg/8zz6mfVT
